### PR TITLE
[TASK] Fix description of additionalCanonicalizedUrlParameters

### LINKED ISSUE #5152: Wrong documentation for $GLOBALS['TYPO3_CONF_VARS']['FE']['additionalCanonicalizedUrlParameters']
The documentation of the setting differs from the official changelog entry https://docs.typo3.org/permalink/changelog:important-86577, the description of the setting in TYPO3 Core itself as well as corresponding code place https://github.com/TYPO3/typo3/blob/main/typo3/sysext/frontend/Classes/Utility/CanonicalizationUtility.php#L40. Instead of excluding, this setting is to add additional parameters to the canonicalized url.

### PR DESCRIPTION
**Pull request by garvinhicking**
The described logic of this is actually the inverse of it.

Closes #5152

Releases: main, 13.4, 12.4

### CODE REVIEW

File: Documentation/ApiOverview/Seo/CanonicalApi.rst [hunk 1]
```diff
@@ -16,27 +16,32 @@ cHash.
...
-It is possible to exclude specific arguments from being appended.
+Including specific arguments from the generation
+================================================
+
+TYPO3 will building a URI of the current page and append query strings
+which are needed for the cHash calculation (vital arguments to uniquely identify
+the given content URI).
+
+It is possible to additionally include specific arguments.
 This is achieved by adding those arguments to a PHP variable:
```
Comment (o-ba) [inline Documentation/ApiOverview/Seo/CanonicalApi.rst:29]: Let's call this configuration, not "just" PHP variable.

File: Documentation/ApiOverview/Seo/CanonicalApi.rst [hunk 2]
```diff
@@ -16,27 +16,32 @@ cHash.
...
-TYPO3 will fallback to building a URL of current page and appending query strings.
-It is possible to exclude specific arguments from being appended.
+Including specific arguments from the generation
+================================================
+
+TYPO3 will building a URI of the current page and append query strings
+which are needed for the cHash calculation (vital arguments to uniquely identify
+the given content URI).
+
+It is possible to additionally include specific arguments.
```
Comment (o-ba) [inline Documentation/ApiOverview/Seo/CanonicalApi.rst:28]: Let's add the "This is especially important with for example detail pages of records. The query parameters are crucial to show the right content." here for clarification.

File: Documentation/ApiOverview/Seo/CanonicalApi.rst (hunk 1 above)
Comment (garvinhicking) [inline Documentation/ApiOverview/Seo/CanonicalApi.rst:29]:
```suggestion
This is achieved by adding those arguments to the configuration:
```

File: Documentation/ApiOverview/Seo/CanonicalApi.rst [hunk 3]
```diff
@@ -16,27 +16,32 @@ cHash.
...
+.. _canonicalapi-additionalparameters:
 
-TYPO3 will fallback to building a URL of current page and appending query strings.
-It is possible to exclude specific arguments from being appended.
+Including specific arguments from the generation
+================================================
+
+TYPO3 will building a URI of the current page and append query strings
+which are needed for the cHash calculation (vital arguments to uniquely identify
+the given content URI).
```
Comment (garvinhicking) [inline Documentation/ApiOverview/Seo/CanonicalApi.rst:26]:
```suggestion
the given content URI). This is especially important with for example detail pages of records. The query parameters are crucial to show the right content.
```

File: Documentation/ApiOverview/Seo/CanonicalApi.rst [hunk 4]
```diff
@@ -16,27 +16,32 @@ cHash.
    EXT:seo. You can find information about how to install and use it in the
    :doc:`EXT:seo manual <ext_seo:Index>`.
 
-Excluding arguments from the generation
-=======================================
+.. _canonicalapi-additionalparameters:
 
-TYPO3 will fallback to building a URL of current page and appending query strings.
-It is possible to exclude specific arguments from being appended.
+Including specific arguments from the generation
```
Comment (o-ba) [inline Documentation/ApiOverview/Seo/CanonicalApi.rst:21]:
```suggestion
Including specific arguments for the URL generation
```

File: Documentation/ApiOverview/Seo/CanonicalApi.rst [hunk 5]
```diff
@@ -18,7 +18,7 @@ cHash.
 
 .. _canonicalapi-additionalparameters:
 
-Including specific arguments from the generation
+Including specific arguments for the URL generation
 ================================================
```
Comment (linawolf) [inline Documentation/ApiOverview/Seo/CanonicalApi.rst]:
```suggestion
Including specific arguments for the URL generation
====================================================
```

File: Documentation/ApiOverview/Seo/CanonicalApi.rst [hunk 6]
```diff
@@ -16,27 +16,32 @@ cHash.
...
 
-Excluding arguments from the generation
-=======================================
+.. _canonicalapi-additionalparameters:
 
-TYPO3 will fallback to building a URL of current page and appending query strings.
-It is possible to exclude specific arguments from being appended.
-This is achieved by adding those arguments to a PHP variable:
+Including specific arguments for the URL generation
+====================================================
```
Comment (linawolf) [inline Documentation/ApiOverview/Seo/CanonicalApi.rst:22]:
```suggestion
Including specific arguments for the URL generation
===================================================
```

**Pull request merged**